task/08-objects-tasks.js

```diff
@@ -27,9 +27,8 @@ function Rectangle(width, height) {
 	this.width = width;
 	this.height = height;
 
-	Rectangle.prototype.getArea = () => this.width * this.height;
-	
 }
+Rectangle.prototype.getArea = () => this.width * this.height;
```

Comment:
We should use the context in the right way (this).
Please see, https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/this

Reply:
I've already done as you said and i get an error in travis. I dont understand why it doesn't work...
![err](https://user-images.githubusercontent.com/64584016/90417100-f7419d00-e0bb-11ea-81e3-a585b2685a7a.png)

Reply:
arrow function is not suitable here.

Reply:
solved) thanks for an explanation!